pl_bolts/models/detection/components/__init__.py

```diff
@@ -0,0 +1 @@
+from pl_bolts.models.detection.components.torchvision_backbones import create_torchvision_backbone
```

Comment:
Do you think there's a possibility to use `TORCHVISION_MODEL_ZOO` in another place in the future? If so, this should be imported in `__init__.py`, but if not, it's fine as is :]
```suggestion
from pl_bolts.models.detection.components.torchvision_backbones import create_torchvision_backbone
from pl_bolts.models.detection.components._supported_models import TORCHVISION_MODEL_ZOO
```

Reply:
Possibly, but let's add when the purpose comes. We shouldn't actually expose this to users to import as it is from torchvision and not from `pl_bolts`.